src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,30 +1,40 @@
-# TO-DO: Complete the selection_sort() function below 
-def selection_sort( arr ):
+# TO-DO: Complete the selection_sort() function below
+def selection_sort(arr):
     # loop through n-1 elements
     for i in range(0, len(arr) - 1):
         cur_index = i
         smallest_index = cur_index
         # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
-
-
+        # (hint, can do in 3 loc)
+        for j in range(cur_index, len(arr)):
+            if arr[j] < arr[smallest_index]:
+                smallest_index = j
 
```

Comment:
Great job using a second for loop. Using the bookshelf analogy, this would be when you've found the first smallest index and are checking all books to the right to see if they are smaller. Awesome, readable implementation.